src/components/app/Overview/Overview.tsx

```diff
@@ -1,50 +1,77 @@
+import AboutAppInfoModal from '../details/AboutAppInfoModal'
 import {
     ExternalLinkIdentifierType,
     ExternalLinksAndToolsType,
     ExternalLinkScopeType,
-} from '../../../externalLinks/ExternalLinks.type'
-import { getExternalLinks, getMonitoringTools } from '../../../externalLinks/ExternalLinks.service'
-import { sortByUpdatedOn } from '../../../externalLinks/ExternalLinks.utils'
-import { AppLevelExternalLinks } from '../../../externalLinks/ExternalLinks.component'
-import './AppOverview.scss'
-import AboutTagEditModal from '../AboutTagEditModal'
+} from '../../externalLinks/ExternalLinks.type'
+import { getExternalLinks, getMonitoringTools } from '../../externalLinks/ExternalLinks.service'
+import { sortByUpdatedOn } from '../../externalLinks/ExternalLinks.utils'
+import { AppLevelExternalLinks } from '../../externalLinks/ExternalLinks.component'
+import './Overview.scss'
+import AboutTagEditModal from '../details/AboutTagEditModal'
 import Tippy from '@tippyjs/react'
-import AppStatus from '../../AppStatus'
-import { StatusConstants } from '../../list-new/Constants'
-import { getModuleInfo } from '../../../v2/devtronStackManager/DevtronStackManager.service'
-import { ModuleStatus } from '../../../v2/devtronStackManager/DevtronStackManager.type'
+import AppStatus from '../AppStatus'
+import { StatusConstants } from '../list-new/Constants'
+import { getModuleInfo } from '../../v2/devtronStackManager/DevtronStackManager.service'
+import { ModuleStatus } from '../../v2/devtronStackManager/DevtronStackManager.type'
+import { toast } from 'react-toastify'
+import { createAppLabels } from '../service'
 
-export default function AppOverview({ appMetaInfo, getAppMetaInfoRes }: AppOverviewProps) {
+export default function AppOverview({ appMetaInfo, getAppMetaInfoRes, isJobOverview }: AppOverviewProps) {
     const { appId } = useParams<{ appId: string }>()
     const [isLoading, setIsLoading] = useState(true)
     const [currentLabelTags, setCurrentLabelTags] = useState<TagType[]>([])
     const [fetchingProjects, projectsListRes] = useAsync(() => getTeamList(), [appId])
     const [showUpdateAppModal, setShowUpdateAppModal] = useState(false)
     const [showUpdateTagModal, setShowUpdateTagModal] = useState(false)
+    const [editMode, setEditMode] = useState(false)
+    const [newDescription, setNewDescription] = useState<string>(appMetaInfo?.description)
     const [externalLinksAndTools, setExternalLinksAndTools] = useState<ExternalLinksAndToolsType>({
         fetchingExternalLinks: true,
         externalLinks: [],
         monitoringTools: [],
     })
-    const [otherEnvsLoading, otherEnvsResult] = useAsync(() => Promise.all([getAppOtherEnvironment(appId), getModuleInfo(ModuleNameMap.ARGO_CD)]), [appId])
+    const [otherEnvsLoading, otherEnvsResult] = useAsync(
+        () => Promise.all([getAppOtherEnvironment(appId), getModuleInfo(ModuleNameMap.ARGO_CD)]),
```

Comment:
environment details are unnecessary for Jobs overview

Reply:
Yes but we are using the same page for apps to render deployment status